### ISSUE TITLE
During certain runs it appears we return early

### ISSUE THREAD

**Issue by shawn-hurley** [konveyor/java-analyzer-bundle#47]
On one run, we get two matches, and in the next, we get one. It appears that this only happens when multiple rules are running. 

I think there may need to be a quick look on the SymbolPRoviders returning early because we do see the search match hittting on both.

**Comment by pranavgaikwad** [comment]
I am moving this to Demo F, we need to close Demo E for release

**Comment by Parthiba-Hazra** [comment]
Hi @shawn-hurley, I want to work on this issue. Can you please provide a bit more information about it.

**Comment by shawn-hurley** [comment]
@pranavgaikwad @jmle I don't think that we are seeing this behavior anymore? If that is the case can you close this issue?

**Comment by dymurray** [comment]
Closing due to inactivity or age, please reopen if you feel this was done in error
